mobile/src/bundles/employer/components/employer-onboarding-form/employer-onboarding-form.tsx

```diff
@@ -6,17 +6,20 @@ import {
     FormField,
     Input,
     PhotoPicker,
-    ScrollView,
-    Text,
     View,
 } from '~/bundles/common/components/components';
 import {
     CountryList,
+    EmployerBottomTabScreenName,
     IconName,
-    TextCategory,
 } from '~/bundles/common/enums/enums';
-import { useAppForm, useCallback } from '~/bundles/common/hooks/hooks';
+import {
+    useAppForm,
+    useCallback,
+    useRoute,
+} from '~/bundles/common/hooks/hooks';
 import { globalStyles } from '~/bundles/common/styles/styles';
+import { LabelForButtonEmployerProfile } from '~/bundles/employer/enums/enums';
 import { type EmployerOnboardingFormDto } from '~/bundles/employer/types/types';
```

Comment:
wrong import for LabelForButtonEmployerProfile
And maybe it would be better to call it something else?

Reply:
Refresh here import for EmployerDataSubmitLabel to clear the conflict <img width="444" alt="image" src="https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/70234990/49dc9483-dee0-4274-8800-25c2562dd165">